src/cli/commands/list.js

```diff
@@ -180,21 +181,36 @@ export function filterTree(tree: Tree, filters: Array<string>): boolean {
   return notDim && (found || hasChildren);
 }
 
+export function getDevDeps(manifest: Object): Set<string> {
+  const devDepSet = new Set();
+  Object.keys(manifest.devDependencies).forEach(key => devDepSet.add(`${key}@${manifest.devDependencies[key]}`));
+  return devDepSet;
+}
+
 export async function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
   const lockfile = await Lockfile.fromDirectory(config.lockfileFolder, reporter);
   const install = new Install(flags, config, reporter, lockfile);
-  const {requests: depRequests, patterns, workspaceLayout} = await install.fetchRequestFromCwd();
+
+  const {requests: depRequests, patterns, manifest, workspaceLayout} = await install.fetchRequestFromCwd();
   await install.resolver.init(depRequests, {
     isFlat: install.flags.flat,
     isFrozen: install.flags.frozenLockfile,
     workspaceLayout,
   });
 
+  let pkgPatterns = [];
```

Comment:
May be a more explanatory name than `pkgPatterns`? May be `usedPatterns` or `activePatterns`?